docs/releasing.md

```diff
@@ -0,0 +1,13 @@
+# Releasing uPortal-home
+
+## Apache release rules
+
+This project has [adopted Apache rules][] with necessary or pragmatic local adaptations.
```

Comment:
I agree with your pragmatic approach to leverage only the rules that make sense for the community.

My concern is leaving it at "necessary or pragmatic local adaptations" will start to build up a list of unspoken and unwritten rules that the community follows.
uPortal as build up some documentation debt in the area of unwritten rules, the lack of knowledge transfer can create a barrier to entry for interested developers.
uPortal Home has the oppurtunity to avoid documentation debt in this area entirely, since the practice are still being established.